pkg/test/common/utils.go

```diff
@@ -15,9 +15,13 @@ import (
 )
 
 const (
-	BaseURL       = "http://127.0.0.1:%s"
-	BaseSecureURL = "https://127.0.0.1:%s"
-	SleepTime     = 100 * time.Millisecond
+	BaseURL               = "http://127.0.0.1:%s"
+	BaseSecureURL         = "https://127.0.0.1:%s"
+	SleepTime             = 100 * time.Millisecond
+	Username              = "test"
```

Comment:
If you ask me these are values used in very specific test cases. Why leak them in a library?
IMHO they shouldn't have been globals in the first place.

Reply:
We have code duplication in a lot of places(including functions that are copy-pasted). If they do not rely on some extensions libraries I think it is better to keep common code under test/common

Reply:
This is not reusable test code, this is test **data**, specific to some tests, which just happens to be repeated.
If there are functions we should move to the libraries, we can look at moving those.

Reply:
Disagree on this. If those are test data than it should be defined (as variables) inside tests. This are constants that we reuse all over again due to some copy-paste and if something is not available in the test package, we copy-paste them and import there. Not a good practice.

Reply:
Then they need to become test variables. The username "test", password "test", repo name `everyone/isallowed` are specific to some tests. There's nothing general enough for these 3 to be package level variables or constants. The only one which could make some sense to be reused is `AuthorizationAllRepos`.

If someone made a bad decision by making them constants, it doesn't mean we should make that worse by moving them to another package and exporting them.

I don't want people to start thinking:
- Do I need to use this test.Username value, or can I use any usernames in my new tests? Would my new test break if I use a different username?
- Are there any dependencies in the test library on these variables?
- I want to use a different username, do I need to add it to the package test?

Reply:
Will update the PR by removing almost all constants from every test files and replacing them with local variables.